lib/improver/cli/wxcode.py

```diff
@@ -162,11 +69,12 @@ def process(cubes, wxtree='high_resolution'):
     Returns:
         iris.cube.Cube:
             A cube of weather symbols.
-
     """
-    result = WeatherSymbols(wxtree=wxtree).process(cubes)
-    return result
+    from iris.cube import CubeList
+    from improver.wxcode.weather_symbols import WeatherSymbols
 
+    if not cubes:
+        raise RuntimeError('Not enough input arguments. '
+                           'See help for more information.')
```

Comment:
As this is raising an exception the user still gets a nasty stream of traceback. It also means this CLI behaves differently to all the others (apart from combine) which give basic usage information when invoked without arguments.

Getting consistent behaviour is a bigger job and doesn't need to be done here, but a simple print and return within this if statement would prevent a very off putting response to a novice user.

Reply:
Yep, it's a bit of a nightmare actually - a much bigger piece of work. I think we can probably revisit this once we've got everything ported.